api/models.py

```diff
@@ -14,3 +14,22 @@ def __str__(self):
         """one-line docstring for representing the Profile object."""
         return f'{self.user.first_name}'
 
+
+
+
+
+class add_loan_record(models.Model):
+    loan=models.CharField(max_length=200, null=True)
+    user_id=models.ForeignKey(User,on_delete=models.CASCADE)
+    due_date=models.DateField(auto_now=True)
+    created=models.DateField(auto_now_add=True)
+    amount=models.TextField()
+    interest_rate=models.IntegerField()
+    paid=models.BooleanField(default=False)
+    borrower=models.CharField(max_length=200)
```

Comment:
The borrower field should be a Boolean, because a borrower as well a lender can create loan records.
This field should be a Boolean.